packages/ethersjs-provider/src/EthersProvider.ts

```diff
@@ -5,14 +5,17 @@ import { EthersProvider as EProvider } from "contract-dependencies-ethers";
 import { ethers } from "ethers";
 import { Abi } from "ethereum";
 import * as _ from "lodash";
+import { Web3AsyncSendable } from "./Web3AsyncSendable";
 
 interface ContractMapping {
     [contractName: string]: ethers.utils.Interface;
 }
 
-export class EthersProvider extends ethers.providers.JsonRpcProvider implements Provider, EProvider {
+export class EthersProvider extends ethers.providers.Web3Provider implements Provider, EProvider {
     private contractMapping: ContractMapping = {};
 
+    constructor(web3AsyncSendable: Web3AsyncSendable) { super(web3AsyncSendable); }
```

Comment:
Is this redundant?

Reply:
Oops, yeah. This constructor was originally different, but got changed later. I'll remove it.